spec/stimulus/turbo_spec.rb

```diff
@@ -52,27 +52,36 @@
       fill_in author_field, with: comment.author
       fill_in text_field, with: comment.text
```

Comment:
make sure the text is some unique value

Reply:
When this test runs, we have a fresh database.